asgiref/server.py

```diff
@@ -56,7 +56,7 @@ def run(self):
         """
         Runs the asyncio event loop with our handler loop.
         """
-        event_loop = get_running_loop()
+        event_loop = asyncio.get_event_loop()
```

Comment:
What situation are you trying to improve where yuo want to get an event loop that isn't running, but somehow still run something in it?

Reply:
The using original function ```get_running_loop()``` makes StatelessServer have to work with a running loop. Otherwise, it would cause ```RuntimeError: no running event loop```.   While ```asyncio.get_event_loop()``` can work either loop is running or not.  I do not understand that is there some disadvantage use a loop that is not running and then make it run tasks?